server/routerlicious/packages/lambdas/package.json

```diff
@@ -71,7 +71,7 @@
 		"json-stringify-safe": "^5.0.1",
 		"lodash": "^4.17.21",
 		"nconf": "^0.12.0",
-		"semver": "^6.3.0",
+		"semver": "^7.5.3",
```

Comment:
we should also be updating `@types/semver` here (and probably elsewhere) to 7.5.0.

I would expect this change needs a pnpm-lock change within the server release group as well.

Reply:
OK